src/frontend/calendar/index.js

```diff
@@ -25,11 +22,15 @@ function Calendar( { events } ) {
 		year: currentYear,
 	};
 	const [ { month, year }, setDate ] = useState( currentMonthYear );
-	const [ currentView, setView ] = useState( CALENDAR_VIEW );
-	const isView = ( toMatch ) => currentView === toMatch;
+	const {
+		isCalendarView,
+		isListView,
+		setCalendarView,
+		setListView,
```

Comment:
I think the more general `isView`/`setView` works better for possible future iterations, but I don't feel strongly about it.

Reply:
Yeah, i changed to this because I didn't want to expose the IDs, meaning I didn't think it would be a fun api to have to push the correct view key to `setView` in consumers.  This is pretty verbose though and 3+ views would be ugly.